PSChiaPlotter/Public/Get-ChiaKPlotCombination.ps1

```diff
@@ -1,31 +1,85 @@
 function Get-ChiaKPlotCombination{
+    <#
+        .SYNOPSIS
+            Display compatible combinations of plots to fill a plot.
+        
+        .DESCRIPTION
+            Display compatible combinations of plots to fill a plot.
+        
+        .PARAMETER FreeSpace
+            Specifies the parameter name.
+        
+        .PARAMETER DriveLetter
+            Specifies the extension. "Txt" is the default.
+        
+        .PARAMETER upto
+            Only return combinations with this being the max size.
+            Options: 32, 33, 34 or 35 (default is all combinations)
+    
+        .PARAMETER c
+            Clears the screen first.
+            
+        .INPUTS
+            Inputs (if any)
+                    
+        .OUTPUTS
+            Displays a list of possible plot size combinations to maximize drive space.
+            
+        .EXAMPLE
+            PS> Get-ChiaKPlotCombination -FreeSpace 108.877420954
+            File.txt
+            
+        .EXAMPLE
+            PS> Get-ChiaKPlotCombination
+            File.doc
+            
+        .EXAMPLE
+            PS> Get-ChiaKPlotCombination
+            File.doc
+            
+        .LINK
+            PowerShell Gallery: https://www.powershellgallery.com/packages/PSChiaPlotter
+
+        .LINK
+            GitHub Project: https://github.com/MrPig91/PSChiaPlotter
+
+        .LINK
+            Set-Item
+    #>
     [CmdletBinding(DefaultParameterSetName = "DriveLetter")]
     param(
         [Parameter(ParameterSetName="FreeSpace")]
         [int64[]]$FreeSpace,
         [Parameter(ParameterSetName="DriveLetter")]
-        [string[]]$DriveLetter = (Get-Volume).DriveLetter
+        [string[]]$DriveLetter = (Get-Volume).DriveLetter,
+        [string]$UpTo = "K35",
+        [switch]$c
     )
-
+    if($c) { Clear-Host }
+    $KSFilter = $UpTo
+    if( $KSFilter.Substring(0,1) -ne "K" ) { $KSFilter = "K$UpTo" }
     if ($PSCmdlet.ParameterSetName -eq "FreeSpace"){
         foreach ($space in $FreeSpace){
             $Max = Get-MaxKSize -TotalBytes $space
-            $AllCombos = Get-OptimizedKSizePlotNumbers $Max | sort RemainingBytes
+            $AllCombos = Get-OptimizedKSizePlotNumbers -MaximizedKSize $Max -KSFilter $KSFilter | Sort-Object RemainingBytes
```

Comment:
I think we are missing the updated "Get-OptimizedKSizePlotNumbers" with the parameter -KSFilter. I think that must be a new parameter you created, but have not yet included in this Pull Request.